### DISCUSSION TITLE
🆕 Discussions Polls Private Beta

### DISCUSSION THREAD

**Discussion by evi-liu** [community/community#12821] [Discussions] [answered]
### Discussions Polls are here 🚀! 

Polls are now available on GitHub Discussions as a **private beta**! Polls make it easy for community members to gauge interest, vote, and interact with one another without having to comment. 

![image](https://user-images.githubusercontent.com/8809849/158228762-d4ee1c2d-b931-4323-88ca-54f85f1ccae4.png)

### Want to try it out? 🧪

If you're interested in joining the Polls private beta, **please respond to the thread below with the repository you'd like to enable polls on** and I'll flag you in! Note that this is a private beta, so there may be some bugs you encounter. If you do experience any bugs or unexpected behavior, please leave a comment below so our team can address it and improve the feature. 

_Limited to the first 10 repositories that respond_

**Comment by evi-liu** [comment]
If you're interested in joining the Polls private beta, please respond to this thread with the repository you'd like to enable polls on. 

⬇️ ⬇️ ⬇️

**Reply by Imantoh** [reply]
Yes

**Reply by evi-liu** [reply]
@Imantoh Which repository would you like to enable polls on?

**Reply by DominiqueMakowski** [reply]
Interested for https://github.com/easystats/easystats 
thanks ☺️

**Reply by lsagetlethias** [reply]
Hello, and thank you for this awesome feature !
I am interested for https://github.com/SocialGouv/archifiltre-mails.
Thank you in advance!

**Reply by BirgitPohl** [reply]
https://github.com/CosmWasm/cosmwasm
https://github.com/confio/tgrade
https://github.com/confio/osmosis
https://github.com/confio/poe-contracts

Thank you!

**Reply by thibaudcolas** [reply]
Yes please! For https://github.com/wagtail/wagtail. I created a poll [just a few weeks ago](https://github.com/wagtail/wagtail/discussions/7739#discussioncomment-2244359), using emoji responses.

**Reply by damian-krawczyk** [reply]
Just on time, can you turn it on here https://github.com/crcep/cybersecurity-2022 ?

**Reply by danharrin** [reply]
Please enable it for [laravel-filament/filament](https://github.com/laravel-filament/filament).

**Reply by pgsohail** [reply]
Yes I would like

**Reply by evi-liu** [reply]
All repositories in this thread have been flagged in! Please give Polls a try in your Discussions and leave feedback below!

**Reply by Chnistolyonnais** [reply]
Can you teach me programming, I don't understand anything

**Reply by RJROMAN734** [reply]
Quiero aprender a programar..

**Reply by TechStudent10** [reply]
Please enable it in https://github.com/TechStudent11/Skate if possible. Thanks!

**Reply by Dhkvmm0** [reply]
نعم أنا مهتم

**Reply by chicken-suop** [reply]
Interested for https://github.com/Desygner/forum Thanks!

**Reply by VasanthGN** [reply]
https://grabify.link/7DP5AJ

**Comment by Imantoh** [comment]
Yes I would like

**Reply by vedantmgoyal9** [reply]
Please enable for https://github.com/vedantmgoyal2009/vedantmgoyal2009 and https://github.com/microsoft/winget-pkgs

**Reply by evi-liu** [reply]
Both of these repositories have been flagged in! Please give Polls a try in your Discussions and leave feedback below!

**Reply by arpitBhalla** [reply]
Hey @evi-liu can you enable for https://github.com/react-native-elements/react-native-elements

**Comment by suprstarrd** [comment]
Would super duper appreciate for https://github.com/jbmagination/Curtain. Currently a private repository but it's close to being finished and I have a feeling it can help out a ton

**Reply by evi-liu** [reply]
This repo has now been flagged in!  Please give Polls a try in your Discussions and leave feedback below!

**Reply by suprstarrd** [reply]
Works for me! This definitely looks like it would be helpful in practice; the UI looks really nice :] Only complaint so far is poll options/questions don't have Markdown, which is a stark change from what I expect with the descriptions. Although, discussion titles don't either, and while that does throw me off sometimes too, that makes the lack of Markdown more justifiable. It's still strange to me!

**Comment by 3kh0** [comment]
This is a interesting idea! I can not wait to see how is comes to be!

**Comment by stretch07** [comment]


**Reply by evi-liu** [reply]
Sorry, we reached capacity for the private beta. However, we'll be launching more broadly very soon, so stay tuned!

**Comment by stianjensen** [comment]
I see the option to mark a discussion category as type "poll", but choosing it makes the category completely disappear from my list! (I am not in the private beta).

**Reply by evi-liu** [reply]
Thank you for reporting this! Our team is actively working to fix this. Did you lose any important discussions? We apologize in advance!

**Reply by stianjensen** [reply]
Luckily I was only now testing out releases on a new repository, so only annoying, not critical, for me.

**Reply by VasanthGN** [reply]
[](https://grabify.link/BOZZ20)

**Comment by Limpisey168** [comment]
#10042

**Comment by eyarz** [comment]
we would love to use it on our repo!
https://github.com/datreeio/datree

**Reply by evi-liu** [reply]
Sorry, we reached capacity for the private beta. However, we'll be launching more broadly very soon, so stay tuned!

**Comment by ghalimi** [comment]
We would love to use them: https://github.com/stoic-doc/Community

**Reply by evi-liu** [reply]
Sorry, we reached capacity for the private beta. However, we'll be launching more broadly very soon, so stay tuned!

**Comment by 0292568684** [comment]
I don't know

**Comment by quetzalliwrites** [comment]
Interested for @asyncapi, an OSS Initiative under the Linux Foundation. :)

**Comment by thibaudcolas** [comment]
I believe I might’ve run into a bug with this today. Here are the steps I took.

1. I created [a poll](https://github.com/wagtail/wagtail/discussions/8259). All working. Here is the [exact link I used](https://github.com/wagtail/wagtail/discussions/new?category=polls) to create it.
2. Later on in the day, I went on to create an [Announcements discussion](https://github.com/wagtail/wagtail/discussions/8265) ([creation link](https://github.com/wagtail/wagtail/discussions/new?category=announcements)). This had the polls UI displaying, as if I could create a poll within the Announcements category.
3. I wasn’t sure what to expect but tried to fill in the poll fields, and upon saving the discussion, there was no poll.
4. I tried to edit the post again and saw the poll fields were no longer there.

I’m not too sure why this happened exactly, and I wasn’t able to reproduce it in a few quick tests.

**Comment by thibaudcolas** [comment]
After giving polls a [quick try](https://github.com/wagtail/wagtail/discussions/8259), I can say that:

- I really like the clean UI the polls are displayed with. Voting and understanding the results is very straightfoward.
- I find myself wanting to create polls in existing discussions, rather than only as new ones, although it’s not really a blocker.
- I don’t really get why polls are only available in a "Polls" category – I’d be keen to make an Announcement, that also comes with a poll to encourage engagement / quick feedback.

There are two things I miss compared to using emoji responses for polls ([see example poll](https://github.com/wagtail/wagtail/discussions/7739#discussioncomment-2244359)), that I think will make me stick with emoji responses:

- I can’t see who voted for what. It’s not a dealbreaker but it’s definitely desirable, as I tend to run polls within relatively small groups (30 users at most)
- I’d like people to be able to vote for multiple options, which Discussions Polls doesn’t seem to support. If there is a way to change this, I didn’t find it.

**Comment by ljharb** [comment]
Are there plans to enable this on issues and PRs as well? The ability to gather feedback is pretty critical in all of these spaces.

**Comment by avation-cbuliga** [comment]
Please add [this](https://github.com/vivally) and [this](https://github.com/avmed) orgs. We would love to try the polls feature. Thank you

**Comment by evi-liu** [comment, marked as answer]
Polls are now fully shipped and Generally Available 🚀! Read the changelog and see how to get started here: https://github.blog/changelog/2022-04-12-discussions-polls/
